<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPrompt</class>
 <widget class="QDialog" name="LoginPrompt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="verticalSpacing">
    <number>-1</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="3">
      <widget class="QLabel" name="imgLock">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/services/images/service_login.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="lblLoginInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lblUserName">
       <property name="text">
        <string>User name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="edtUserName"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lblPassword">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="edtPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QCheckBox" name="chbStaySigned">
       <property name="text">
        <string>Stay signed in (only during this session)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="chbRememberPassword">
       <property name="text">
        <string>Remember my password</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogin">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>Confirm and close the window</string>
       </property>
       <property name="accessibleName">
        <string>Ok button</string>
       </property>
       <property name="accessibleDescription">
        <string>Confirm and close the window</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>Discard and close window</string>
       </property>
       <property name="accessibleName">
        <string>Cancel button</string>
       </property>
       <property name="accessibleDescription">
        <string>Discard and close window</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>LoginPrompt</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLogin</sender>
   <signal>clicked()</signal>
   <receiver>LoginPrompt</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
